CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Ráhpajávvre is the Lule Sámi spelling of Rapaselet.
 
 - [#1246](https://github.com/nf-core/sarek/pull/1246) - Back to dev
 - [#1259](https://github.com/nf-core/sarek/pull/1259) - nf-prov plugin
+- [#1252](https://github.com/nf-core/sarek/pull/1252) - Added NGSCheckMate tool for checking that samples come from the same individual
```

Comment:
I think this may be in the wrong section, nf-prov was already released. you might need to merge in dev.

Reply:
And also can you sort it by ascending numbers? oh god @maxulysse has pulled me to the dark side 😆